packages/sit-onyx/src/components/OnyxNavItem/future/OnyxNavItem.vue

```diff
@@ -0,0 +1,75 @@
+<script setup lang="ts">
+const props = defineProps<{ active?: boolean }>();
+</script>
+<template>
+  <li
+    :class="{
+      'onyx-future-nav-item': true,
+      'onyx-future-nav-item--active': props.active,
+    }"
```

Comment:
```suggestion
    class="onyx-future-nav-item" 
    :class="{ 'onyx-future-nav-item--active': props.active }"
```

Reply:
I refactored the component and it's using the list item internally now. We don't have the classes any more :)